doc/palette.md

```diff
@@ -1,30 +1,43 @@
 # Palette
 
-Throughout your game you are going to need to use colors in lots of places. There are two classes on `dart:ui` that can be used, `Color` and `Paint`.
+Throughout your game you are going to need to use colors in lots of places. There are two classes on
+`dart:ui` that can be used, `Color` and `Paint`.
 
-The `Color` class is nothing but a wrapper over a single simple ARGB color in hexadecimal integer format. So to create a Color object you just need to pass in the color as an integer in the ARGB format.
+The `Color` class is nothing but a wrapper over a single simple ARGB color in hexadecimal integer
+format. So to create a `Color` object you just need to pass in the color as an integer in the ARGB
+format.
 
-You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully opaque green (the 'mask' would be `0xAARRGGBB`). Do note the first two hexadecimal digits are for the alpha channel (transparency), unlike on regular (non-A) RGB. The max(FF = 255) means fully opaque.
+You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully
+opaque green (the 'mask' would be `0xAARRGGBB`). Do note the first two hexadecimal digits are for
+the alpha channel (transparency), unlike on regular (non-A) RGB. The max(FF = 255) for the two first
+digits means fully opaque, and the min (00 = 0) means fully transparent.
 
 In the material flutter package there is a color enum to make it easy to use common colors:
 
 ```dart
-import 'package:flutter/material.dart' as material;
+import 'package:flutter/material.dart';
 
-Color color = material.Colors.black;
+Color black = Colors.black;
 ```
 
-Some more complex methods might also take a Paint, which is a more complete option that allows you to define more options related to stroke, colors, filters, blends. However, normally when using even the more complex APIs, you want just an instance of a Paint representing just a single simple plain solid color.
+Some more complex methods might also take a `Paint` object, which is a more complete option that
+allows you to define more options related to stroke, colors, filters and blends.
+However, normally when using even the more complex APIs, you just want an instance of a `Paint`
```

Comment:
Maybe add a `**Note**:` that we recommend that you dont constantly create new `Paint` objects each render frame?

Reply:
Done